src/locales/en-US.json

```
@@ -239,6 +239,7 @@
     "weightsPct": "Weights percent",
     "willBurn": "This operation will burn",
     "withdraw": "Withdraw",
+    "withdrawalAmt": "Withdrawal amt",
```

We can add a "." here for the abbreviation 
```suggestion
    "withdrawalAmt": "Withdrawal amt.",
```
We could also remove the amount and just show "Withdraw", i think it's still comprehensible and save some space.

---

How about "My Balance" and "Withdrawal Amt"? Then it's balanced and grammatical

---

Could also just be "Withdrawal" I guess